samples/bookinfo/policy/productpage_envoy_ratelimit.yaml

```diff
@@ -0,0 +1,88 @@
+apiVersion: networking.istio.io/v1alpha3
```

Comment:
I think this can be removed now?

Reply:
added it, so that this can be used to write the task in istio.io as they use bookinfo for examples..